ddpui/api/transform_api.py

```
@@ -751,3 +765,618 @@ def post_unlock_canvas(request, payload: LockCanvasRequestSchema):
+# V2 CRUD operations for CanvasNode
+@transform_router.post("/v2/dbt_project/models/{dbtmodel_uuid}/nodes/")
+@has_permission(["can_create_dbt_model"])
+def post_create_src_model_node(request, dbtmodel_uuid: str):
+    """
+    Create a new CanvasNode representing a source/model node.
+
+    Simplifies the complex source/model creation by using unified CanvasNode.
+    """
+    orguser: OrgUser = request.orguser
+
+    warehouse = OrgWarehouse.objects.filter(org=orguser.org).first()
```

Ruff (0.14.5)

Redefinition of unused `warehouse` from line 15

(F811)